pkg/olm/clusterserviceversion.go

```diff
@@ -3,6 +3,7 @@ package olm
 import (
 	"context"
 	"fmt"
+
```

Comment:
Can you remove this space?

Reply:
it is impossible to remove this space, once I remove it I'm getting  error:
"pkg/lso/localvolumediscovery.go:5: File is not `goimports`-ed (goimports)
level=info msg="File cache stats: 102 entries of total size 674.9KiB"
	"fmt"
"
and, after I fix it with goimport tool it return it back